SECURITY.md

aryanguptacsvtu opened the pull request:
Hi @RayBreeze 
I've added the `SECURITY.md` as per the structure suggested in the issue...
Please let me know if any changes are needed.😊

aryanguptacsvtu committed:
SECURITY.md

aryanguptacsvtu commented:
Hi @RayBreeze , if everything looks good to you , could you please merge my PR...😊

RayBreeze reviewed (changes requested):
Hi @aryanguptacsvtu ,
We need some small changes which includes removing the emojis and a bit more specific security policy as per the software. Please make these changes and I will be happy to review it ☺️. Thanks for your patience.

Regards,
Samman

aryanguptacsvtu committed:
Update SECURITY.md

Updated the Security Policy document to improve clarity and organization, including sections on supported versions, vulnerability reporting, responsible disclosure, and security best practices.

aryanguptacsvtu commented:
@RayBreeze  , I have made the required changes as per your need.
Please review it once..😊

RayBreeze reviewed (approved):
Hello @aryanguptacsvtu,
Thank you for your contribution in SysMood. These small documentation change will surely help us for better maintenance of the project for me and the fellow contributors. Thanks for making SysMood better. Please don't forget to add a star and watch for future releases. Love to see more contributions in future...🙂

Regards,
Samman
